README.md

```diff
@@ -265,7 +265,7 @@ Inflector.ordinalize(-1021)  // => '-1021st'
 
 ```js
 Inflections inflections([string locale])
-inflections([string locale], [function(Inflections) fn])
+Inflections inflections([string locale], [function(Inflections) fn])
```

Comment:
Thank-you @martinandert, I've removed that commit now. So only my commit involving a change to the Makefile exists now.